jsparty/js-party-11.md

```diff
@@ -186,7 +186,7 @@ It's not an open standard, so that means you can't freely implement this, if tha
 
 So \[unintelligible 00:21:40.07\] doesn't implement this, because it's not free as in free stuff, and so...
```

Comment:
Can you update this to "Gecko" while you're in the file?

Reply:
Yea sure :)